### PR TITLE
created class GallonUsToLitreConverter

### PR DESCRIPTION
**Pull request by Paddigton**

### CODE REVIEW

**Review by friwarto — CHANGES_REQUESTED** [review]
The converter is missing! There are only changes to `Program.java` in this Pull Request.

File: Units/Program.java [hunk 1]
```diff
@@ -105,7 +105,7 @@ public static void main(String[] args) {
 
 			case 18:
 				conv = new HorsepowerToWattConverter();
-			break;
+			break;*/
```
Comment (friwarto) [inline Units/Program.java]: wuts dis

**Review by friwarto — CHANGES_REQUESTED** [review]
Please remove code comment.

**Comment by Paddigton** [comment]
aront noł :D

**Pull request merged**